docs/user/openstack/known-issues.md

```diff
@@ -165,3 +165,13 @@ $ rm -f openshift/99_openshift-cluster-api_worker-machineset-*.yaml
 ```
 [bugzilla-upi]: https://bugzilla.redhat.com/show_bug.cgi?id=1927244
 [manifests-removal]:../openstack/install_upi.md#remove-machines-and-machinesets
+
+## Unable to create external load-balancers using pre-defined FIP (In-tree provider)
+
+Installations with in-tree cloud provider enabled does support usage of
+a Floating IP address that was pre-defined in the Service specification
+because of [a Neutron policy restriction][bugzilla-in-tree]. To create
```

Comment:
Well, you can still use it, just make sure to change the policy and allow the user to create FIPs with specified address. It'd be real good to ask Neutron folks on why it's disallowed by default. I'd guess there's some security consideration here and we could explain it.

Reply:
You're right. I'm gonna ask them

Reply:
Just found the steps to update that policy https://access.redhat.com/solutions/6069071